autotest/pyscripts/test_gdal_retile.py

```diff
@@ -278,7 +335,5 @@ def test_gdal_retile_cleanup():
                 pass
 
     shutil.rmtree('tmp/outretile4')
-
-
-
+    shutil.rmtree('tmp/outretile5')
```

Comment:
should be conditionalized by `if os.path.exists('tmp/outretile5')` (or something similar) to avoid an exception when the test is skipped